.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove the .vscode folder from this PR? 
You can still keep it in your local repo `echo ".vscode/" >> .gitignore`